helm/templates/api/api-configmap.yaml

```diff
@@ -534,6 +533,60 @@ data:
     cloud:
     {{- toYaml .Values.cloud | nindent 6 -}}
     {{- end }}
+    {{- if .Values.api.federation.enabled }}
+    cluster:
+      type: hazelcast
+      hazelcast:
+        config-path: ${gravitee.home}/config/hazelcast.xml
+    cache:
+      type: hazelcast
+      hazelcast:
+        config-path: "${gravitee.home}/config/hazelcast.xml"
+    {{- end }}
+    {{- end }}
```

Comment:
This extra `end` replaces the end removed a few lines above, right ?

Reply:
yes it was misplaced before. If a user defines `.Values.api.auth` the Helm deployment would failed...